packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -338,8 +346,15 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
                 relay.realizedLpFeePct + depositData.slowRelayFeePct + depositData.instantRelayFeePct,
                 depositData.amount
             );
-
-        l1Token.safeTransferFrom(msg.sender, depositData.l1Recipient, depositData.amount - feesTotal);
+        // If the L1 token is WETH then: a) pull WETH from instant relayer b) unwrap WETH c) send ETH to recipient.
+        uint256 recipientAmount = depositData.amount - feesTotal;
+        if (isWethPool) {
+            l1Token.safeTransferFrom(msg.sender, address(this), recipientAmount);
+            WETH9Like(address(l1Token)).withdraw(recipientAmount);
+            depositData.l1Recipient.transfer(recipientAmount);
```

Comment:
Just FYI, we should remember that `transfer` only sends along 2300 gas, which could cause reverts if calling into a contract that wants to do something on the incoming transfer. If we don't expect deep integrations here, I think this is fine. Just something to be aware of.